src/main/kotlin/no/nav/familie/ef/mottak/task/SendPåminnelseOmDokumentasjonsbehovTilDittNavTask.kt

```diff
@@ -73,26 +83,31 @@ class SendPåminnelseOmDokumentasjonsbehovTilDittNavTask(
         stønadType: StønadType?,
     ) =
         harSendtInnNySøknad(søknad, søknader) ||
-            harSendtInnEttersendingIEttertid(søknad, ettersendinger) ||
-            stønadType == null ||
-            tilhørendeBehandleSakOppgaveErPåbegynt(søknad, stønadType)
+                harSendtInnEttersendingIEttertid(søknad, ettersendinger) ||
+                stønadType == null ||
+                tilhørendeBehandleSakOppgaveErPåbegynt(søknad, stønadType)
 
     private fun harSendtInnNySøknad(søknad: Søknad, søknader: List<Søknad>): Boolean =
         søknader.filter { it.id != søknad.id }
             .filter { SøknadType.hentSøknadTypeForDokumenttype(it.dokumenttype) != SøknadType.OVERGANGSSTØNAD_ARBEIDSSØKER }
             .any { it.opprettetTid > søknad.opprettetTid }
 
-    private fun harSendtInnEttersendingIEttertid(søknad: Søknad, ettersendinger: List<Ettersending>): Boolean =
-        ettersendinger.any { it.opprettetTid > søknad.opprettetTid }
+    private fun harSendtInnEttersendingIEttertid(søknad: Søknad, ettersendinger: List<Ettersending>): Boolean {
+        dokumentasjonsbehovvarslingerBrukerSendtInn.increment()
+        return ettersendinger.any { it.opprettetTid > søknad.opprettetTid }
+    }
 
-    private fun tilhørendeBehandleSakOppgaveErPåbegynt(søknad: Søknad, stønadType: StønadType) =
-        saksbehandlingClient.kanSendePåminnelseTilBruker(
+    private fun tilhørendeBehandleSakOppgaveErPåbegynt(søknad: Søknad, stønadType: StønadType): Boolean {
+        dokumentasjonsbehovvarslingerSaksbehandlerStartet.increment()
```

Comment:
Samme her. Denne inkremmenteringen vil bli gjort for hver gang man sjekker mot ef-sak om søknaden har ført til en påbegynt behandleSak oppgave.